.gitignore

```diff
@@ -25,5 +25,6 @@ testbin/*
 
 *.qcow2
 neonvm/hack/vmlinuz
+neonvm/hack/kernel/vmlinuz
```

Comment:
I'm torn whether to leave the old gitignore rule or not, but would lean more towards removing it and notifying people in slack.
Thoughts?

Reply:
I'll create a follow-up PR which will remove old paths (we also have one in `make kernel` job), we can merge it in a month, for example

Reply:
Why wait? What happens if we remove it now?